src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Unix/KeyAgreeRecipientInfoPalOpenSsl.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Pkcs;
+
+namespace Internal.Cryptography.Pal.OpenSsl
+{
+    internal sealed class KeyAgreeRecipientInfoPalOpenSsl : KeyAgreeRecipientInfoPal
+    {
+        public override DateTime Date
+        {
+            get
+            {
+                throw new NotImplementedException();
```

Comment:
Are these cases of just not being implemented yet, or will they never be implemented?  If the former, this is the right exception type.  If the latter, we should throw PlatformNotSupportedException instead.